understanding/20/pause-stop-hide.html

```diff
@@ -1,107 +1,85 @@
 <!DOCTYPE html>
 <html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
-   <meta charset="UTF-8"></meta>
+   <meta charset="UTF-8"/>
    <title>Understanding Pause, Stop, Hide</title>
    <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
 </head>
 <body>
    <h1>Understanding Pause, Stop, Hide</h1>
-   
+
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
          <dt>Goal</dt><dd>Fewer users are distracted by content that updates or moves.</dd>
          <dt>What to do</dt><dd>Let users control content changes that occur in parallel with other content.</dd>
-         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.</dd> 
+         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate when they are distracted by continuous movement.</dd>
```

Comment:
Trying to keep these short. How about:
```suggestion
         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits are distracted by continuous movement.</dd>
```